web/js/codeworld_shared.js

```diff
@@ -168,62 +249,10 @@ function registerStandardHints(successFunc)
     codeworldKeywords['main'] = 'deprecated';
     codeworldKeywords['program'] = 'builtin';
 
-    lines = lines.sort().filter(function(item, pos, array) {
-        return !pos || item != array[pos - 1];
-    });
-
-    var hintBlacklist = [
-        // Symbols that only exist to implement RebindableSyntax or map to
-        // built-in Haskell types.
-        "Bool",
-        "IO",
-        "fail",
-        "fromCWText",
-        "fromDouble",
-        "fromInt",
-        "fromInteger",
-        "fromRational",
-        "fromString",
-        "ifThenElse",
-        "toCWText",
-        "toDouble",
-        "toInt",
-
-        // Deprecated exports.
-        "MousePress",
-        "MouseRelease",
-        "MouseMovement",
-        "MouseButton",
-        "LeftButton",
-        "RightButton",
-        "MiddleButton",
-        "Maybe",
-        "Nothing",
-        "Just",
-        "withDefault",
-        "hasValue",
-        "definitely",
-        "path",
-        "thickPath",
-        "text",
-        "styledText",
-
-        // Experimental exports.
-        "White",
-        "Black",
-        "Gray",
-        "Grey",
-        "Red",
-        "Orange",
-        "Yellow",
-        "Green",
-        "Blue",
-        "Purple",
-        "Pink",
-        "Brown",
-        "Pi",
-    ];
+    codeWorldBuiltinsDocs['main'] = createHover('main :: Program', 0, 4, 'Your program.');
```

Comment:
I don't think it's important to add docs for deprecated code styles.  We can leave out `main`

Reply:
Ok, done.